docs/Connection.rst

```diff
@@ -125,11 +125,9 @@ Creating a ``Connection`` using the ``REST`` tranport.
    const connection = new Connection({
      transport: 'rest',
      transportOptions: {
-        host: 'myhost',
         database: '*LOCAL',
         username: 'myuser',
         password: 'mypass'
-        port: 80,
-        path:'/cgi-bin/xmlcgi.pgm',
+        url: 'http://myhost:80/cgi-bin/xmlcgi.pgm',
```

Comment:
```suggestion
        url: 'http://myhost.example.com/cgi-bin/xmlcgi.pgm',
```

port 80 is the default port for http, no need to specify it. Also example.com is specifically reserved for examples in documentation: https://www.iana.org/domains/reserved